src/components/ScholarshipAmountField.test.jsx

```diff
@@ -12,13 +14,19 @@ const formik = {
 
 function renderWithAmountType(type) {
   formik.values.amount.type = type;
-  return render(<ScholarshipAmountField formik={formik} />);
+  return render(
+    <Suspense fallback="loading">
+      <I18nextProvider i18n={i18n}>
+        <ScholarshipAmountField formik={formik} />
+      </I18nextProvider>
+    </Suspense>
+  );
 }
 
-test('renders select options', () => {
+test('renders select options', async () => {
   renderWithAmountType(AmountType.Fixed);
 
-  const select = screen.getByRole('button');
+  const select = await waitFor(() => screen.getByRole('button'));
```

Comment:
I think you can do this:
```suggestion
  const select = await screen.findByRole('button');
```

Don't know if I forgot to suggest this elsewhere in other PRs.

Reply:
I tried that and it failed the tests because the only things rendered is `loading`

Reply:
Weird, `findByRole` _should_ work.